src/backends/simple.ts

```diff
@@ -0,0 +1,442 @@
+import type {
+    DecodedFrames,
+    Packet,
+    Demuxer,
+    Decoder,
+    Filterer
+} from '@antoinemopa/beamcoder';
+import beamcoder from '@antoinemopa/beamcoder';
+import type { ImageData } from 'canvas';
+import { createImageData } from 'canvas';
+import { BaseExtractor } from '../BaseExtractor';
+import type { Extractor, ExtractorArgs, InterpolateMode } from '../../framefusion';
+import { DownloadVideoURL } from '../DownloadVideoURL';
+
+const LOG = false;
```

Comment:
I loved that previously we had more granularity on the logs. depending on what I was debugging, I would turn specific flags on & off.

Can we bring this back?

Reply:
I can also do it the next time I debug. Maybe it will turn out to not be necessary.

Reply:
So far, I just turned it on/off. Maybe because this version is smaller, we don't have as many logs? I'll leave it up to you to determine in future PR's @antoineMoPa

Reply:
Yeah, that's a detail, I don't want this to be in the way of merging the PR